src/tests/functional/shared_test_classes/src/single_op/generate_proposals.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_op/generate_proposals.hpp"
+
+namespace ov {
+namespace test {
+std::string GenerateProposalsLayerTest::getTestCaseName(
+        const testing::TestParamInfo<GenerateProposalsTestParams>& obj) {
+    std::vector<InputShape> shapes;
+    ov::op::v9::GenerateProposals::Attributes attributes;
+    ElementType model_type;
```

Comment:
I suggest to not use `ElementType` because it's enum alias, and it has no methods like `to_string()` or `get_type_name()`, and I'm not sure, does it have `operator<<` support to be used in `getTestCaseName`

Reply:
done